pulser-core/pulser/waveforms.py

```diff
@@ -533,7 +533,7 @@ class RampWaveform(Waveform):
     Args:
         duration: The waveform duration (in ns).
         start: The initial value (in rad/µs).
-        stop: The final value (in rad/µs).
+        stop: The final value at duration - 1 (in rad/µs).
```

Comment:
How about
```suggestion
        stop: The value (in rad/µs) at the final sample. 
```
And also change `start` accordingly.